ova/provision.sh

```diff
@@ -1,56 +1,129 @@
+  sed -i "s/wazuh_kibana-[0-9\.]\+_[0-9\.]\+/wazuh_kibana-${WAZUH_VERSION}_${ELK_VERSION}/g" ${INSTALLER}
+fi
+
+# Execute unattended installer
+sh ${INSTALLER}
+
+# Remove installer
+rm ${INSTALLER}
+
+# Check kibana status
+until [[ "$(curl -u admin:admin -XGET https://localhost/status -I -s -k | grep HTTP)" == *"200"* ]]; do
+    echo "Waiting for Kibana..."
+    sleep 2
+done
+
+# Stop services and enable manager
+systemctl stop kibana filebeat elasticsearch wazuh-manager
+systemctl enable wazuh-manager
 
-# Setting wazuh default root password
+# Disable all host modules
+sed -i "s/<disabled>no/<disabled>yes/g" /var/ossec/etc/ossec.conf
+sed -i "s/<enabled>yes/<enabled>no/g" /var/ossec/etc/ossec.conf
 
-cp ${libraries_files}/automatic_set_ram.sh ${automatic_set_ram_location}
-chmod +x "${automatic_set_ram_location}/automatic_set_ram.sh"
-echo "@reboot . /etc/automatic_set_ram.sh" >> ram_cron
+# Disable /var/log/ files and commands monitoring
+sed -i "s/<localfile>/<!--localfile>/g" /var/ossec/etc/ossec.conf
+sed -i "s/<\/localfile>/<\/localfile-->/g" /var/ossec/etc/ossec.conf
 
+# Enable ossec_auth modules
+sed -i '/<auth>/ {N; s/<auth>.*yes/<auth>\n\ \ \ \ <disabled>no/g}' /var/ossec/etc/ossec.conf
 
-crontab ram_cron
-rm -rf ram_cron
+# Custom Login Page
+# Edit window title
+sed -i "s/null, \"Elastic\"/null, \"Wazuh\"/g" /usr/share/kibana/src/core/server/rendering/views/template.js
 
-yes wazuh | passwd root
-hostname wazuhmanager
+# Download custom files (background, logo and template)
+curl -so /vagrant/custom_welcome.tar.gz https://wazuh-demo.s3-us-west-1.amazonaws.com/custom_welcome_opendistro_docker.tar.gz
+tar -xf /vagrant/custom_welcome.tar.gz -C /vagrant
 
-# Ssh config
-sed -i "s/PasswordAuthentication no/PasswordAuthentication yes/" /etc/ssh/sshd_config
-echo "PermitRootLogin yes" >> /etc/ssh/sshd_config
+# Copy necesaries files
+cp /vagrant/custom_welcome/wazuh_logo_circle.svg /usr/share/kibana/src/core/server/core_app/assets/
+cp /vagrant/custom_welcome/wazuh_wazuh_bg.svg /usr/share/kibana/src/core/server/core_app/assets/
+cp /vagrant/custom_welcome/template.js.hbs /usr/share/kibana/src/legacy/ui/ui_render/bootstrap/template.js.hbs
 
-# OVA Welcome message
-cat > /etc/issue << EOF
+# Add custom configuration to css
+less /vagrant/assets/customWelcomeKibana.css >> /usr/share/kibana/src/core/server/core_app/assets/legacy_light_theme.css
 
-Welcome to the Wazuh OVA version $WAZUH_VERSION
-Access the Wazuh Web Interface at https://\4{eth0}
+# Get actual RAM of machine and split it in half
+ram=$(( $(free -m | awk '/^Mem:/{print $2}') / 2 ))
+
+# Change de jvm.options with the new RAM use
+sed -i "s/^-Xms[0-9]\+[gm]/-Xms${ram}m/" /etc/elasticsearch/jvm.options
+sed -i "s/^-Xmx[0-9]\+[gm]/-Xmx${ram}m/" /etc/elasticsearch/jvm.options
+
+# Remove vagrant user - by default this script runs in /home/vagrant 
```

Comment:
Over-commenting

Reply:
Fixed in https://github.com/wazuh/wazuh-packages/pull/746/commits/6fb50e3584df557334da525324a73a5c32ae0551